site/code/index.html.js

```diff
@@ -299,8 +299,82 @@ Code that executes a hello world GraphQL query with \`graphql-clj\`:
 
 ### C# / .NET
 
-  - [graphql-dotnet](https://github.com/graphql-dotnet/graphql-dotnet): GraphQL for .NET
-  - [graphql-net](https://github.com/ckimes89/graphql-net): Convert GraphQL to IQuerable
+[graphql-dotnet](https://github.com/graphql-dotnet/graphql-dotnet)
+
+A .NET library that provides a GraphQL implementation with .NET Core support.
+
+Code that executes a hello world GraphQL query with \`graphql-dotnet\`:
+
+\`\`\`csharp
```

Comment:
This is because the markdown is inlined in a JavaScript multiline string template literal so have to escape the markdown code block syntax.  See line 20.